### DISCUSSION TITLE
File uploads in Settings

### DISCUSSION THREAD

**Discussion by armin-salihovic** [area17/twill#2461] [Q&A]
## Description

According to #1238, file uploads in the settings section should already be working. 

### Steps to reproduce

1. Add a file field in the settings section
2. Upload a file and save

### Expected result

Saved file.

### Actual result

Nothing happens.

### Versions

Twill version: 3.0.2
Laravel version: 10.12
PHP version: 8.1
Database engine: MySQL 8

**Comment by dassudip2001** [comment]
hi @armin-salihovic I can interest to fix this issue plz can you help me how to set up local env

setups like a normal Laravel application or any other process

**Comment by pauldwight** [comment]
@armin-salihovic did you include the $fileParams into your block_editor config?

```php
return [
    'block_editor' => [
        'crops' => [
            ...
        ],
        'files' => ['single_file']
    ]
``` 

```php
<x-twill::files
    name="single_file"
    label="Single file"
    note="Add one file (per language)"
/>
```
